spec/outputs/elasticsearch_spec.rb

```diff
@@ -900,6 +900,34 @@ def mock_actions_with_response(*resp)
         end
       end
     end
+
+    context "when using http protocol" do
+      context "when using ssl without cert verification" do
```

Comment:
not necessary but better categorizes the test. This is all in the hopes that more tests will come and the outer context will have other contexts/tests.